### DISCUSSION TITLE
Openedai speech not working

### DISCUSSION THREAD

**Discussion by yuchenxie4645** [open-webui/open-webui#5261] [General]
I followed the directions for installing it with CUDA GPU using docker compose (option 1). Everything was working and openedai-speech-server-1 had shown up oin the container ID in docker stats. However, THE PROBLEM IS: OpenWEBUI Keeps telling me "server connection error" How do I fix it. 
![image](https://github.com/user-attachments/assets/d0cb6fdf-a4a6-48e5-92c2-2037b90f9475)
![image](https://github.com/user-attachments/assets/ed66183a-963f-4064-9480-c1c1bf90bb68)
![image](https://github.com/user-attachments/assets/0706a97e-354c-471f-a503-3ad0ca4b46ca)

**Comment by silentoplayz** [comment]
Could you try changing the `TTS` model in your STT Settings from `tts-1-hd` to `tts-1`?

**Reply by yuchenxie4645** [reply]
I will try that right now.

**Reply by yuchenxie4645** [reply]
It still does not work, it still says server connection error.

**Comment by yuchenxie4645** [comment]
I found the solution I checked the port of the docker container using docker port openedai-speech-server-1 and found out that it was hosted on 0.0.0.0:8000, so instead of host.docker.internal:8000/v1 I put my_ip:8000/v1. It worked like a charm.
